f600233d1b154d507b782f84604bcb12/update/sys_script_include_b29002b097a211106fa8b4b3f153afc4.xml

```diff
@@ -725,25 +744,77 @@ Igce.prototype = {
             }
             var toolsUsed = "";
             if (igceInstructions.how_est_dev_tools_used) {
-                toolsUsed += igceInstructions.how_est_dev_tools_used;
+                toolsUsed = igceInstructions.how_est_dev_tools_used;
+                // Fix leading comma
+                if (toolsUsed.substring(0, 1) === ",") {
+                    toolsUsed = toolsUsed.substring(1);
+                }
+                toolsUsed = toolsUsed
+                    .replace("AWS", "AWS pricing calculator")
+                    .replace("GOOGLE_CLOUD", "Google Cloud pricing calculator")
+                    .replace("MICROSOFT_AZURE", "Microsoft Azure pricing calculator")
+                    .replace("ORACLE_CLOUD", "Oracle Cloud pricing calculator")
+                    .replace("PREVIOUSLY_PAID_PRICES", "Prices previously paid on a same or similar contract")
+                    .replace("OTHER", igceInstructions.how_est_dev_other_tools_used);
+
+                if (igceInstructions.how_est_dev_other_tools_used) {
+                    toolsUsed = toolsUsed
+                        .replace("OTHER", igceInstructions.how_est_dev_other_tools_used);
+                }
             }
-            if (igceInstructions.how_est_dev_other_tools_used) {
-                toolsUsed += " " + igceInstructions.how_est_dev_other_tools_used;
+            var previousEstimateComparison = "";
+            if (currentEnvironment && currentEnvironment.current_environment_exists === "Yes") {
```

Comment:
Not sure if UI changed what is being send over, but should this be all caps (`YES`)? If it is working okay then ignore this.